src/libraries/System.Text.Json/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,3 @@
+Compat issues with assembly System.Text.Json:
+CannotRemoveAttribute : Attribute 'System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute' exists on 'System.Text.Json.Serialization.JsonConverterAttribute.ConverterType' in the implementation but not the reference.
```

Comment:
Why is this necessary? can't we add it to the ref?

Reply:
Good catch. I think I did it for downlevel tfm's but I'll double check.

I might have to do something similar to what I did in the Extensions project to import the attribute cs file into the ref to define them as internal since they're not available in downlevel tfms.

Reply:
Yeah, we should be consistent here between Extensions and other downlevel projects.

I originally didn't put the attributes in the Extensions `ref` assembly because they are `internal`. But thinking more about this, if we ever get an analyzer for the linker we would need those attributes (even though they are internal) in the ref assemblies.

---
In reply to: [464612345](https://github.com/dotnet/runtime/pull/40185#discussion_r464612345) [](ancestors = 464612345)

Reply:
Yeah. I added these cause I saw it was specifically excluded for non-net5.0 targets in the ref: 
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Text.Json/ref/System.Text.Json.cs#L521

However, it seems it was because these attributes are not available, so I'll update to what I did in Extensions.

Reply:
Fixed. Thanks.